app/views/articles/_actions.html.erb

```diff
@@ -76,10 +77,14 @@
             href="https://www.facebook.com/sharer.php?u=<%= article_url(@article) %>">
             Share to Facebook
           </a>
+        </div>
+
+        <web-share-wrapper shareurl="<%= article_url(@article) %>" sharetitle="<%= @article.title %>" sharetext="<%= @article.description %>" template="web-share-button">
         </web-share-wrapper>
         <template id="web-share-button">
-          <a href="#" class="dropdown-link-row crayons-link crayons-link--block">Share Post</a>
+          <a href="#" class="dropdown-link-row crayons-link crayons-link--block">Share Post via...</a>
```

Comment:
From an accessibility POV this should really be a button, as it executes an action on the current page, rather than linking anywhere.

In saying that, if we make it a button we'd need to consider the design a bit more, as it would/should look different than the links.

I guess what I'm saying is I would _like_ to say this is a blocking issue, but I appreciate it might not be super-straightforward to just change 😅